examples/wasm-emscripten/src/main.rs

```diff
@@ -0,0 +1,111 @@
+#![no_main]
+
+// Embed models into the .wasm file.
+#[derive(rust_embed::RustEmbed)]
+#[folder = "models/"]
+pub struct Models;
```

Comment:
Since this only embeds one file I'd rather this be an `include_bytes!` so we can forego the `rust_embed` dependency